### ISSUE TITLE
How to specify jump-host

### ISSUE THREAD

**Issue by ansible75013** [czerwonk/junos_exporter#104]
How we can use this exporter when we can only access our infrastructure through a jump-host. ?
We can specify that into ~/.ssh/config but how to say that to the exporter ?

**Comment by 4xoc** [comment]
I don't think this is possible at the moment. The crypto/ssh library that takes care of the SSH connection doesn't support the use of a config file or jumphost at all for that matter. It's not a vanilla openssh but native go.

**Comment by Bischoff** [comment]
Any reason why @martintopholm's commit is not integrated upstream?

**Comment by martintopholm** [comment]
I think I just never did a pull-request. Not using the feature myself, I wasn't sure it was sufficient to solve this issue.
